.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @paketo-buildpacks/stacks-maintainers
```

Comment:
I think we should add the steering committee as well. In a perfect world, the Steering committee wouldn't need to get involved, but steering committee members can vote and approve changes. This is important now, being short-staffed.

Reply:
Added it.